src/components/CountryData.js

```diff
@@ -0,0 +1,70 @@
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router';
+import { useSelector, useDispatch } from 'react-redux';
+import { fetchData } from '../redux/covidData/covid';
+
+function CountryData() {
+  const covidReducer = useSelector((state) => state.covidReducer.covid);
+  const dispatch = useDispatch();
+  const { name } = useParams();
+  const findCountry = covidReducer.find((country) => country.country === name);
+  useEffect(() => {
+    if (!covidReducer.length) {
+      dispatch(fetchData());
+    }
+  }, []);
+  const {
+    todayCases,
+    todayDeaths,
+    todayRecovered,
+    country,
+    tests,
+    deaths,
+    flag,
+    population,
+  } = findCountry;
+  return (
+    <div className="data">
+      <div className="data-flag">
+        <img src={flag} alt="#" />
+        <h2>
+          Country:
+          {country}
+        </h2>
+      </div>
+      <ul className="list" key={findCountry.country}>
+        <li>
+          <p>
+            Population:
+            { population }
+          </p>
+          <p>
+            People Tested Today :
+            { tests }
+          </p>
+          <p>
+            New Cases:
+            { todayCases }
+          </p>
+          <p>
+            Deaths:
+            { deaths }
+          </p>
+          <p>
+            Dead Today:
+            { todayDeaths }
+          </p>
+          <p>
+            People Recovered Today:
+            { todayRecovered }
+          </p>
+
+        </li>
+
+      </ul>
+    </div>
```

Comment:
- [ ] We are trying to build a mobile app. That means that our app should maintain responsiveness across multiple mobile screen versions. Once I reduce the screen width to less then 441px it starts to overflow. Kindly fix this.

- [ ] Additionally, this page continues with a white background once the properties are exhausted. The app background color should cover the entire screen for better UI.

  ![ando](https://user-images.githubusercontent.com/81468675/188886119-e337ad2a-ea72-457b-82f4-d656ad09b939.jpg)